ng-dev/pr/rebase/index.ts

```diff
@@ -99,9 +99,18 @@ export async function rebasePr(prNumber: number): Promise<number> {
      * Additional flags to perform the autosquashing are added when the user confirm squashing of
      * fixup commits should occur.
      */
-    const [flags, env] = squashFixups
-      ? [['--interactive', '--autosquash'], {...process.env, GIT_SEQUENCE_EDITOR: 'true'}]
-      : [[], undefined];
+
+    let flags: string[] = [];
+    let env = undefined;
+
+    if (squashFixups || interactive) {
+      env = {...process.env, GIT_SEQUENCE_EDITOR: 'true'};
+      flags.push('--interactive');
```

Comment:
do we always want to be interactive when squashing?

Reply:
It was set to that before for autosquash and I just didn't want to break existing functionality.

Reply:
Weirdly thats what we had previously

Reply:
I think that's because the Git sequence editor was never opened. It was a "silent" interactive rebase to activate autosquashing (it doesn't work in a non-interactive rebase; or didn't back then at least)

Reply:
That's a good question. I assumed it was needed, but I will try it locally and make sure.

Reply:
Yeah, it is not needed. I'll put up a quick fix.

Reply:
The fix is in #2728. I should have tested this before, but...I couldn't remember the command to run it locally. LOL